dev/tasks/python-wheels/github.osx.amd64.yml

```diff
@@ -34,8 +34,8 @@ env:
 
 jobs:
   build:
-    name: Build wheel for OS X
-    runs-on: macos-10.15
+    name: Build wheel for macOS
+    runs-on: macos-latest
```

Comment:
Why not `macos-11` explicitly?

Reply:
Because I don't think that we need exactly `macos-11` for this. We specify `MACOSX_DEPLOYMENT_TARGET` explicitly for this job. So I think that we can generate logically same binaries regardless of macOS version.
If this doesn't depend on exactly `macos-11` and we use `macos-latest` here, we don't need to update this again when `macos-11` is deprecated.

Should we use `macos-11` explicitly here? If so, why do we need exactly `macos-11` here?